_includes/configuration/table_background_color_map.md

```diff
@@ -2,7 +2,7 @@
 
 {{site.requires_5_9v}}
 
-This option is used to specify the default values for the table cell background color picker `tablecellbackgroundcolor`. If no values are defined, the toolbar button and menu item will use the values or default values of the [`color_map` option]({{site.baseurl}}/configure/content-appearance/#color_map). This option accepts Hex, RGB and HSL color values.
+This option is used to specify the default values for the table cell background color picker, which can be opened with the `tablecellbackgroundcolor` toolbar button or menu item. If no values are defined, the toolbar button and menu item will use the values or default values of the [`color_map` option]({{site.baseurl}}/configure/content-appearance/#color_map). This option does not modify the background color picker in the table dialog. This option accepts Hex, RGB and HSL color values.
```

Comment:
```suggestion
This option is used to specify the default values for the table cell background color picker, which can be opened with the `tablecellbackgroundcolor` toolbar button or menu item. If no values are defined, the toolbar button and menu item will use the values or default values of the [`color_map` option]({{site.baseurl}}/configure/content-appearance/#color_map). This option does not currently modify the background color picker in the table dialogs. This option accepts Hex, RGB and HSL color values.
```
Maybe? Just to highlight it's all dialogs and it's not really intentional and will be changed later.

Reply:
Updated both